lib/registration.pm

```diff
@@ -197,7 +198,13 @@ sub add_suseconnect_product {
 
     my $try_cnt = 0;
     while ($try_cnt++ <= $retry) {
-        eval { assert_script_run("SUSEConnect $debug_flag -p $name/$version/$arch $params", timeout => $timeout); };
+        if (is_transactional) {
+            # Currently only sle-micro6.0/6.1 support Extras, but there is SCC setup issue which is not sorted yet in 6.1
+            # We need to wait till it is fixed
```

Comment:
I think it is already fixed in 6.1 https://openqa.suse.de/tests/15444679#step/suseconnect_scc/51